packages/components/src/ui/icon/test/__snapshots__/component.js.snap

```diff
@@ -0,0 +1,96 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`props should render correctly 1`] = `
+.emotion-0.emotion-0.emotion-0.emotion-0.emotion-0.emotion-0.emotion-0 {
```

Comment:
Is this something that is applied for every component or it's only shared and can be removed from snapshots? I see it everywhere.

Reply:
I think these come from the base `View` component.

Any styles applied to the component will end up in `emotion-0` as that's the deserialized name for the styles.

For example, if I had a snapshot test like this:

```
const { container } = render( <View css={ { width: '200px' } } /> );
expect( container.firstChild ).toMatchSnapshot();
```

The snapshot would include `width: 200px` in `emotion-0`.

Some of this will be cleaned up if we remove automatic prefix generation as suggested in #21745